bootstrapvz/plugins/minimize_size/tasks/shrink.py

```diff
@@ -70,6 +70,8 @@ class ShrinkVolumeWithQemuImg(Task):
     @classmethod
     def run(cls, info):
         tmp_name = os.path.join(info.workspace, 'shrunk.' + info.volume.extension)
-        log_check_call(
-            ['qemu-img', 'convert', '-O', info.volume.extension, info.volume.image_path, tmp_name])
+        shrink_cmd = ['qemu-img', 'convert', '-O', info.volume.extension, info.volume.image_path, tmp_name]
+        if info.volume.extension == 'qcow2':
+            shrink_cmd.insert(4, '-c')
```

Comment:
I think you need to add some explanation for what this does `insert(4, '-c')` might be a bit... terse :-)

Too bad qemu-img doesn't have long options.